tests/Integration/Schema/Directives/CountDirectiveDBTest.php

```diff
@@ -367,7 +367,7 @@ public function testResolveRelationItemsAndCount(): void
         /** @var User $user */
         $user = factory(User::class)->create();
 
-        /** @var Collection<Task> $tasks */
+        /** @var Collection<int, Task> $tasks */
```

Comment:
fqcn or avoid

Reply:
So also use FQCN for inline `@var`? And this is a new rule? Because I see a lot of inline docs without FQCN (example: 2 lines above this issue).

Reply:
The code is not perfect yet. Generally prefer assert, and always use fqcn in phpdocs.